diff -ruN orig/lib/xdg/telegramdesktop.desktop new/lib/xdg/telegramdesktop.desktop
--- orig/lib/xdg/telegramdesktop.desktop	2022-02-08 10:18:14.000000000 -0600
+++ new/lib/xdg/telegramdesktop.desktop	2022-02-18 11:26:22.126598915 -0600
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=1.5
+Version=1.0
 Name=Telegram Desktop
 Comment=Official desktop version of Telegram messaging app
 TryExec=telegram-desktop
@@ -12,7 +12,6 @@
 MimeType=x-scheme-handler/tg;
 Keywords=tg;chat;im;messaging;messenger;sms;tdesktop;
 Actions=Quit;
-SingleMainWindow=true
 X-GNOME-UsesNotifications=true
 X-GNOME-SingleWindow=true
 
